api/swagger.yaml

```diff
@@ -1577,6 +1577,20 @@ definitions:
           Settable: null
           Value: []
 
+  ObjectVersion:
+    description: |
+      The version number of the swarm related object such as node, service, etc. The version number is
+      needed to avoid conflicting writes. It is required that the client sends the version number along
+      with the modified specification for swarm related objects updates.
```

Comment:
Avoid the passive voice: "The client must send the version number along with the modified specification when updating these objects."